components/edit_post_modal/edit_post_modal.tsx

```diff
@@ -126,51 +189,67 @@ class EditPostModal extends React.PureComponent {
             this.setState({editText: ':' + emojiAlias + ': '});
         } else {
             const {editText} = this.state;
-            const {firstPiece, lastPiece} = splitMessageBasedOnCaretPosition(this.state.caretPosition, editText);
+            const {firstPiece, lastPiece} = splitMessageBasedOnCaretPosition(
+                this.state.caretPosition,
+                editText,
+            );
 
             // check whether the first piece of the message is empty when cursor
             // is placed at beginning of message and avoid adding an empty string at the beginning of the message
-            const newMessage = firstPiece === '' ? `:${emojiAlias}: ${lastPiece}` : `${firstPiece} :${emojiAlias}: ${lastPiece}`;
-
-            const newCaretPosition = firstPiece === '' ? `:${emojiAlias}: `.length : `${firstPiece} :${emojiAlias}: `.length;
-
-            const textbox = this.editbox.getInputBox();
-
-            this.setState({
-                editText: newMessage,
-                caretPosition: newCaretPosition,
-            }, () => {
-                Utils.setCaretPosition(textbox, newCaretPosition);
-            });
+            const newMessage =
+        firstPiece === '' ?
+            `:${emojiAlias}: ${lastPiece}` :
+            `${firstPiece} :${emojiAlias}: ${lastPiece}`;
+
+            const newCaretPosition =
+        firstPiece === '' ?
+            `:${emojiAlias}: `.length :
+            `${firstPiece} :${emojiAlias}: `.length;
+
+            const textbox = this.editbox && this.editbox.getInputBox();
+
+            this.setState(
+                {
+                    editText: newMessage,
+                    caretPosition: newCaretPosition,
+                },
+                () => {
+                    Utils.setCaretPosition(textbox, newCaretPosition);
+                },
+            );
         }
 
         this.setState({showEmojiPicker: false});
 
         if (this.editbox) {
             this.editbox.focus();
         }
-    }
+    };
 
-    handleGifClick = (gif) => {
+    handleGifClick = (gif: string) => {
         if (this.state.editText === '') {
             this.setState({editText: gif});
         } else {
-            const newMessage = ((/\s+$/).test(this.state.editText)) ? this.state.editText + gif : this.state.editText + ' ' + gif;
+            const newMessage = (/\s+$/).test(this.state.editText) ?
+                this.state.editText + gif :
+                this.state.editText + ' ' + gif;
             this.setState({editText: newMessage});
         }
         this.setState({showEmojiPicker: false});
-        this.editbox.focus();
-    }
+        if (this.editbox) {
+            this.editbox.focus();
```

Comment:
There's a shorthand for this null check:
```suggestion
            this.editbox?.focus();
```